test/extensions/filters/http/common/fuzz/uber_filter.cc

```diff
@@ -67,6 +73,13 @@ void UberFilterFuzzer::fuzz(
         proto_config, factory_context_.messageValidationVisitor(), factory);
     // Clean-up config with filter-specific logic before it runs through validations.
     cleanFuzzedConfig(proto_config.name(), message.get());
+    if (with_main_event_loop_) {
```

Comment:
I think that due to the static definition of the UberFilterFuzzer object, once these are set they won't be cleared.
Executing `VerifyAndClear` when resetting should remove these (but need to add the relevant ON_CALLs that are typically done once in the c'tor).